src/combobox/combobox.js

```diff
@@ -178,6 +179,7 @@ function Combobox<OptionT>(props: PropsT<OptionT>) {
     setSelectionIndex(-1);
     setTempValue(value);
     if (onBlur) onBlur(event);
+    if (onFocus) onFocus(event);
```

Comment:
I think this should be in the `handleFocus` function above.

Reply:
Yeah good call out. Updated